docs/pages/beta/crm/crm-beta.yaml

```diff
@@ -179,10 +179,10 @@ paths:
                   type: array
                   items:
                     type: string
-                    enum: [accounts, deals, leads, notes, tasks, events, eventAttendees, calls, emails]
                   description: |
                     Array of associations to include in the response.
-                    OwnerUserId is always included.
+                    Allowed values: accounts, deals, leads, notes, tasks, events, eventAttendees, calls, emails. 
```

Comment:
Just confirming here that we've verified the list of associations allowed for each object?

Also, are you planning to fix the examples in a separate pr or the same pr?

Reply:
Yeah, will post a follow up in the slack thread - but technically these lists are correct as of now. 

Also which example?

Reply:
The example responses  for the associations field
https://docs.vessel.dev/pages/beta/crm/api-reference/get-all-contacts
<img width="504" alt="image" src="https://github.com/vesselapi/all-api-docs/assets/12945091/a56d53f5-314f-472c-a9c1-fb2176864833">